src/components/activity/ArgumentsCellRenderer.svelte

```diff
@@ -0,0 +1,96 @@
+<script lang="ts">
+  import type { ActivityDirective, ActivityType } from '../../types/activity';
+  import type { ValueSchema } from '../../types/schema';
+  import { convertUsToDurationString } from '../../utilities/time';
+
+  export let data: ActivityDirective;
+  export let activityTypes: ActivityType[];
+
+  function formatParameterValue(value: any, schema: ValueSchema): string {
+    if (value === null || value === undefined) {
+      return '';
+    }
+
+    switch (schema.type) {
+      case 'duration':
+        try {
+          return convertUsToDurationString(value, true);
+        } catch (error) {
+          return String(value);
+        }
+
+      case 'series':
+        if (Array.isArray(value)) {
+          if (value.length === 0) {
+            return '[]';
+          } else {
+            return `${value.map(String).join(', ')}`;
+          }
+        }
+        return String(value);
+
+      case 'struct':
```

Comment:
Not sure this is working as intended? Try this with the `ParameterTest` activity in Banananation. Seeing a lot of `[object  Object'`.

<img width="404" height="543" alt="Image" src="https://github.com/user-attachments/assets/edbff747-d0d3-44d6-a499-bc54ea0878fa" />